### PR TITLE
docs: Added documentation for checksum check of downloaded kluctl binary file

### PR DESCRIPTION
**Pull request by matzegebbe**
# Description

Added documentation for checksum check of downloaded kluctl binary file

## Type of change

- [x] New feature (non-breaking change which adds functionality)

All Submissions:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
* [x] I have performed a self-review of my code
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] All commits are signed off which certify that you created the patch and that you agree to the [Developer Certificate of Origin](https://developercertificate.org/)

### CODE REVIEW

File: content/en/getting-started/installation.md [hunk 1]
```diff
@@ -27,6 +27,22 @@ All of these tools must be in your PATH, so that kluctl can easily invoke them.
...
+2. **Optional**: Validate the kluctl binary *only available as of version >= v2.7.0*
+
+   Download the kluctl checksum file:
+   ```bash
+   kluctl_version=$(curl --silent -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kluctl/kluctl/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/checksums.txt"
+   ```
+   Validate the kluctl binary against the checksum file:
+   ```
+   echo $(grep kluctl-linux-amd64 checksums.txt) | sha256sum --check
```
Comment (AljoschaP) [inline content/en/getting-started/installation.md:39]: This line doesn't work for me. Could you check, if there is something missing. @matzegebbe 

Output of the checksums.txt
```
0df45073be51080031febf93afc2cf95f96c68910a8ca5e2db6ed1775412bd1d  kluctl_2.7.7_Linux_arm64.tar.gz
7013966f55c6fb46f109fc1916b4b49f84b0fb66eb1dab283fa0c641615a49c9  kluctl_2.7.7_Darwin_arm64.tar.gz
7c2c6d2ad0b3629323f6d637ccf91ee70d3ad55753b59997536bcb69241f2ec5  kluctl_2.7.7_Windows_x86_64.tar.gz
bdc7235c706dd0dabb3af87f5d12821f89de35903a00aa36d78191fa457d7610  kluctl_2.7.7_Linux_x86_64.tar.gz
f7ae9fecb6b55c142b538e621685780f902a9ea6e0b74113524a8d71f2a1ac7d  kluctl_2.7.7_Darwin_x86_64.tar.gz
```

**Comment by matzegebbe** [comment]
kluctl-v2.7.7 has tar files instead of binarys. I will update the docs in this MR

**Comment by codablock** [comment]
FYI, https://github.com/kluctl/www-kluctl.io/pull/31 introduced an install.sh script to make installation via shell easier. This script also checks the checksum automatically.

File: content/en/getting-started/installation.md [hunk 2]
```diff
@@ -24,7 +24,28 @@ All of these tools must be in your PATH, so that kluctl can easily invoke them.
    ```bash
    # To download a specific version, replace the kluctl_version with the version you need for example kluctl_version={{< param "fullversion" >}}
    kluctl_version=$(curl --silent -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kluctl/kluctl/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl-linux-amd64"
+   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl_${kluctl_version:1}_Linux_x86_64.tar.gz"
+   ```
+   {{< alert >}}
+   Since version `kluctl-v2.7.6` we package the releases in `tar.gz` archive. 
+   If you download older versions of kluctl, the command to check the checksum must be changed to
```
Comment (AljoschaP) [inline content/en/getting-started/installation.md:31]: The part after `[...] changed to` is missing.

File: content/en/getting-started/installation.md [hunk 3]
```diff
@@ -24,7 +24,28 @@ All of these tools must be in your PATH, so that kluctl can easily invoke them.
    ```bash
    # To download a specific version, replace the kluctl_version with the version you need for example kluctl_version={{< param "fullversion" >}}
    kluctl_version=$(curl --silent -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kluctl/kluctl/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl-linux-amd64"
+   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl_${kluctl_version:1}_Linux_x86_64.tar.gz"
```
Comment (AljoschaP) [inline content/en/getting-started/installation.md:27]: Maybe we should add a note somewhere that this is an example for linux amd64 or even better try to make this switchable for each os/arch. For example like golang does it https://go.dev/doc/install :innocent: .

**Pull request closed without merging**